docs/dependencies.md

```diff
@@ -1,32 +1,28 @@
+# Managing Dependencies
 
-# Managing Server-Side Dependencies
+## Managing Server-Side Dependencies
 
-TEAMMATES uses Gradle to manage dependencies to external Java libraries.
+- Dependency management tool: Gradle
+- Library source: [JCenter](https://bintray.com/bintray/jcenter)
+- Configuration: `build.gradle`
 
-- [Adding new libraries](#adding-new-libraries)
-- [Updating libraries](#updating-libraries)
-
-## Adding new libraries
```

Comment:
Isn't this section still applicable?

Reply:
It is, but I don't see the point of flashing it out. A look at `build.gradle` or `package.json` and one should know what to do with the dependencies, or at least read up on the relevant resources somewhere else.